python/pyspark/ml/regression.py

```diff
@@ -708,7 +710,7 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeModel(JavaModel):
+class DecisionTreeModel(JavaModel, JavaPredictionModel):
```

Comment:
@holdenk what about `LinearRegressionModel`, `GeneralizedLinearRegressionModel` (though `numFeatures` is missing on the Scala side), `GBTRegressionModel`, `RandomForestRegressionModel`?